deps/build.jl

```diff
@@ -34,6 +34,10 @@ if is_apple()
     provides(Homebrew.HB, "homebrew/science/hdf5", hdf5, os=:Darwin)
 end
 
+if Sys.KERNEL === :FreeBSD
```

Comment:
`is_bsd()` is true for any BSD-based system, including macOS, OpenBSD, DragonFly, etc. The `BSDPkg` provider only applies to FreeBSD.